content-model/articles.json

```diff
@@ -96,6 +96,32 @@
               }
             }
           },
+          "iframe" : {
+            "type" : "Slice",
+            "fieldset" : "Iframe",
```

Comment:
Nitpicking `iframe`?

Reply:
Figured we'd gone with capped up elsewhere